internal/ir/xds.go

```diff
@@ -231,6 +229,25 @@ func (h HTTPListener) Validate() error {
 // TLSListenerConfig holds the configuration for downstream TLS context.
 // +k8s:deepcopy-gen=true
 type TLSListenerConfig struct {
+	// Certificates contains the set of certificates associated with this listener
+	Certificates []TLSCertificate `json:"certificates,omitempty" yaml:"certificates,omitempty"`
+	// MinVersion defines the minimal version of the TLS protocol supported by this listener.
+	MinVersion *egv1a1.TLSVersion `json:"minVersion,omitempty" yaml:"version,omitempty"`
```

Comment:
recommend using non `egv1a1` types in here

Reply:
Changed.